.gitignore

```diff
@@ -136,3 +136,6 @@ dmypy.json
 
 # Poetry local config
 poetry.toml
+
+# dummy test file
```

Comment:
Is this still relevant?

Reply:
Whoops, didn't mean to leave that in. Good catch, reverted in [300a089](https://github.com/supabase-community/postgrest-py/pull/111/commits/300a089ac19bcb0b1cbfec13a7429b46ca51f5ed)